examples/00_tutorials/plot_single_subject_single_run.py

```diff
@@ -336,41 +357,41 @@
 # Performing an F-test
 # --------------------
 #
-# "active vs rest" is a typical t test: condition versus
-# baseline. Another popular type of test is an F test in which one
-# seeks whether a certain combination of conditions (possibly two-,
-# three- or higher-dimensional) explains a significant proportion of
-# the signal.  Here one might for instance test which voxels are well
-# explained by the combination of the active and rest condition.
+# "listening > rest" is a typical t test: condition versus baseline.
+# Another popular type of test is an F test in which
+# one seeks whether a certain combination of conditions
+# (possibly two-, three- or higher-dimensional)
+# explains a significant proportion of the signal.
+# Here one might for instance test which voxels are well
+# explained by the combination of more active or less active than rest.
```

Comment:
I think it would be good to mention that F-contrasts produce only positive beta values and t-statistics here.